en/sensor/rangefinders.md

```diff
@@ -1,183 +1,189 @@
-# Distance Sensors (Rangefinders)

```

Comment:
This got all stuffed by me fixing the merge commit. I'm can't merge. I'll do my own copy and close this.

NOTE, it is better to create branches from master and merge those :-)